src/taipy/gui-core/GuiCoreLib.py

```diff
@@ -34,15 +35,26 @@ class GuiCore(ElementLibrary):
     "on_scenario_create": ElementProperty(PropertyType.function, f"{GuiCoreContext._VAR_NAME}.create_new_scenario()"),
     "core_changed": ElementProperty(PropertyType.broadcast, GuiCoreContext._CORE_CHANGED_NAME)
   })}
+  
+
 
   def get_name(self) -> str:
     return "taipy_gui_core"
 
   def get_elements(self) -> t.Dict[str, Element]:
-    return GuiCore.__elts
+    return GuiCore.elts
 
   def get_scripts(self) -> t.List[str]:
-    return ["lib/taipy-gui-core.js"]
+    return ["src/taipy/gui-core/lib/taipy-gui-core.js"]
   
   def on_init(self, gui: Gui) -> t.Optional[t.Tuple[str, t.Any]]:
     return GuiCoreContext._VAR_NAME, GuiCoreContext(gui, Core())
+
+page = """
+# Getting started with example
+
+<|taipy_gui_core.scenario_selector|show_add_button|display_cycles|>
+
+"""    
+Gui.add_library(GuiCore())
+Gui(page).run(port=8000, use_reloader=True)
```

Comment:
This should go in a separate demo-....py file (at the root directory of the project)